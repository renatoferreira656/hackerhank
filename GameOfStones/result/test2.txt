2.0 - 0.0
First
11.0 - 0.0
First
2.2 - 1.0
First
6.8 - 4.0
11.333333333333334 - 1.0
First
18.4 - 2.0
30.666666666666668 - 2.0
46.0 - 0.0
First
