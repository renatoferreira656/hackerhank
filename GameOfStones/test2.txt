5
10
55
11
34
92
