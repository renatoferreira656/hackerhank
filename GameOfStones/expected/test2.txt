First
First
First
First
Second
